ops/pebble.py

```diff
@@ -627,19 +645,37 @@ def wait_change(
         raise TimeoutError(
             'timed out waiting for change {} ({} seconds)'.format(change_id, timeout))
 
-    def add_layer(self, layer: Union[str, dict, Layer]):
-        """Dynamically add a layer to the Pebble setup."""
+    def add_layer(self, label: str, layer: Union[str, dict, Layer], combine: bool = False):
+        """Dynamically add a new layer onto the Pebble configuration layers.
+
+        If combine is False (the default), append the new layer as the top
+        layer with the given label. If combine is True, combine the new layer
+        with the layer that has the given label; combining is done according
+        to Pebble's override rules.
```

Comment:
Same text as for the earlier comment, so the remark applies here as well.

Reply:
Done.